From 225bd90e9f52dce60ca362a57f280c85bc986b4a Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Mon, 24 Feb 2020 17:35:53 +0000
Subject: [PATCH 2/5] net: qrtr: fix spelling mistake "serivce" -> "service"

There is a spelling mistake in a pr_err message. Fix it.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 13ef6ae8c0d96c00476ab9c81671e39a9f714e2b)
Signed-off-by: hzy <hzyitc@outlook.com>
---
 net/qrtr/ns.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/qrtr/ns.c b/net/qrtr/ns.c
index a19d33cbaf85e..4ef47eae0bc26 100644
--- a/net/qrtr/ns.c
+++ b/net/qrtr/ns.c
@@ -150,7 +150,7 @@ static int service_announce_del(struct sockaddr_qrtr *dest,
 
 	ret = kernel_sendmsg(qrtr_ns.sock, &msg, &iv, 1, sizeof(pkt));
 	if (ret < 0)
-		pr_err("failed to announce del serivce\n");
+		pr_err("failed to announce del service\n");
 
 	return ret;
 }
-- 
2.25.1

